tests/templates/bootstrap-session.yaml

```diff
@@ -0,0 +1,17 @@
+# Contains the bootstrap configuration for the session instance of the integration tests.
+# The session instance persists over test runs and is used to speed-up the integration tests.
+cluster-config:
+  network:
+    enabled: true
+  dns:
+    enabled: true
+  ingress:
```

Comment:
I guess we can strip this down a bit. We don't need to enable the local-storage, gateway, loadbalancer, ingress and even metrics-server for this test, do we?